phpcs.xml

```diff
@@ -139,7 +139,6 @@
     <rule ref="PEAR.WhiteSpace.ScopeClosingBrace"/>
     <rule ref="Squiz.WhiteSpace.CastSpacing"/>
     <rule ref="Squiz.WhiteSpace.LogicalOperatorSpacing"/>
-    <rule ref="Squiz.WhiteSpace.OperatorSpacing"/>
```

Comment:
Yep, this is a PITA as it doesn't like the ternary operator split onto multiple lines.  It's supposed to ensure that there is a space between operator and operand (which we want to enforce) but seems buggy with expressions over multiple lines.  We should probably push for a fix in PHPCS so that we can still use it for single-line expressions.

Reply:
Can this be a separate PR, and an issue report against PHPCS?

Reply:
I have reported the issue now: https://github.com/squizlabs/PHP_CodeSniffer/issues/3377

Will create a separate PR in a minute.